### PR TITLE
update namespaces for filtres

### PR DESCRIPTION
**Pull request by Kutuzovska**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: src/Data/Reader/Filter/All.php [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Yii\Cycle\Data\Reader\Processor;
+namespace Yiisoft\Yii\Cycle\Data\Reader\Filter;
```
Comment (vjik) [inline src/Data/Reader/Filter/All.php:5]:
```suggestion
namespace Yiisoft\Yii\Cycle\Data\Reader\FilterHandler;
```

File: src/Data/Reader/Filter/All.php [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 
 use Cycle\ORM\Select\QueryBuilder;
 
-final class All extends GroupProcessor
+final class All extends Group
```
Comment (vjik) [inline src/Data/Reader/Filter/All.php:9]:
```suggestion
final class AllHandler extends Group
```

For consistency with https://github.com/yiisoft/data/blob/master/src/Reader/Iterable/FilterHandler/AllHandler.php

And same to other filter handlers.

File: src/Data/Reader/Filter/QueryBuilderFilter.php [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Yii\Cycle\Data\Reader\Processor;
+namespace Yiisoft\Yii\Cycle\Data\Reader\Filter;
 
-interface QueryBuilderProcessor
+interface QueryBuilderFilter
```
Comment (vjik) [inline src/Data/Reader/Filter/QueryBuilderFilter.php:7]:
```suggestion
interface QueryBuilderFilterHandler
```

**Comment by samdark** [comment]
@Kutuzovska would you please adjust names as @vjik suggested?

File: src/Data/Reader/Filter/All.php (hunk 1 above)
Comment (Kutuzovska) [reply src/Data/Reader/Filter/All.php]: Why change namespace special one filter?

File: src/Data/Reader/Filter/All.php (hunk 2 above)
Comment (Kutuzovska) [reply src/Data/Reader/Filter/All.php]: Is a Iterable handler, but I rename simple filter

**Pull request closed without merging**